prowler/providers/aws/services/lightsail/lightsail_instance_public/lightsail_instance_public.py

```diff
@@ -0,0 +1,27 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.lightsail.lightsail_client import lightsail_client
+
+
+class lightsail_instance_public(Check):
+    def execute(self) -> Check_Report_AWS:
+        findings = []
+
+        for arn_instance, instance in lightsail_client.instances.items():
+            report = Check_Report_AWS(self.metadata())
+            report.region = instance.region
+            report.resource_id = instance.id
+            report.resource_arn = arn_instance
+            report.resource_tags = instance.tags
+            report.status = "FAIL"
+            report.status_extended = f"Instance '{instance.name}' has public access"
+            if instance.public_ip == "" and not any(
+                port.access_type == "public" for port in instance.ports
```

Comment:
I cannot see the attribute `access_type` in the Port class

Reply:
https://github.com/prowler-cloud/prowler/pull/3919/files#diff-1ccedd3473e8d0ba377e274a6434e22ecfddb9cc831f2c1e3aa0996a3a90b2beR165